<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="KernelTimeSelect">
     <item>
      <property name="text">
       <string>GPS Time*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UTC Time*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Local (System) Time</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Reference Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="KernelReferenceTime">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="date">
        <date>
         <year>2017</year>
         <month>5</month>
         <day>4</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>1752</year>
         <month>11</month>
         <day>14</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>yyyy:MM:dd hh:mm:ss.zzz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Camera Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="KernelCameraTime">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="date">
        <date>
         <year>2017</year>
         <month>5</month>
         <day>4</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>1752</year>
         <month>11</month>
         <day>14</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>yyyy:MM:dd hh:mm:ss.zzz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>* Requires Internet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ModalSaveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ModalCancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
